app/models/dialog_field_serializer.rb

```diff
@@ -17,7 +17,11 @@ def serialize(dialog_field, all_attributes = false)
     }
 
     if dialog_field.dynamic?
-      extra_attributes["values"] = dialog_field.extract_dynamic_values
+      if dialog_field.kind_of?(DialogFieldSortedItem)
+        extra_attributes["values"] = dialog_field.extract_dynamic_values
```

Comment:
stupid nit but we could make this prettier, this if, something about 
         extra_attributes["values"] = dialog_field.extract_dynamic_values = if/else/end whatever

Reply:
how about something like:

```ruby
key_to_update = dialog_field.kind_of?(DialogFieldSortedItem) ? "values" : "default_value"

extra_attributes[key_to_update] = dialog_field.extract_dynamic_values
```